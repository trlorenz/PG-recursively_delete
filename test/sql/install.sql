CREATE EXTENSION recursively_delete VERSION '0.1.5';

SELECT extversion
FROM pg_catalog.pg_extension
WHERE extname = 'recursively_delete';
