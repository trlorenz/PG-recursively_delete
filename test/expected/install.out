CREATE EXTENSION recursively_delete VERSION '0.1.5';
SELECT extversion
FROM pg_catalog.pg_extension
WHERE extname = 'recursively_delete';
 extversion 
------------
 0.1.5
(1 row)

